test/unit/helpers/peersManager.js

```diff
@@ -0,0 +1,246 @@
+'use strict';
+
+var _ = require('lodash');
+var chai = require('chai');
+var expect = require('chai').expect;
+var sinon = require('sinon');
+var randomPeer = require('../../common/objectStubs').randomPeer;
+var Peer = require('../../../logic/peer');
+
+var peersManager = require('../../../helpers/peersManager');
+
+describe('PeersManager', function () {
+
+	var validPeer;
+
+	beforeEach(function () {
+		validPeer = new Peer(randomPeer);
+		peersManager.peers = {};
+		peersManager.addressToNonceMap = {};
+		peersManager.nonceToAddressMap = {};
+	});
+
+	describe('constructor', function () {
+
+		it('should have empty peers map after initialization', function () {
```

Comment:
All those properties will always be empty because we overwrite them in `beforeEach`